example/empty.cpp

```diff
@@ -1,4 +1,3 @@
-#include "example_unicode_utils.h"
```

Comment:
Was the file renaming intentional here?

Reply:
Yes -- the cmake `configure_file` didn't interpolate anything as far as I could tell

Reply:
Yes, there is nothing to interpolate indeed.

The unobvious reason why it there is `empty.in.cpp` in the repo and not `empty.cpp` is to allow this workflow
- CMake creates `empty.cpp` 
- The `empty.cpp` is available to contributors and users as playground for experiments, to modify, build and debug as part of the nanodbc project.
- The `empty.cpp` is automatically ignored, so contributors don't have to be cautious to not to commit their experiments to the nanodbc repo or forks

Reply:
Thanks @nimish 
If there are better ways to achieve similar, then please feel free to propose a change.

Reply:
> Thanks @nimish
> If there are better ways to achieve similar, then please feel free to propose a change.

Honestly, not sure -- this does seem like a decent solution. It's documented in a comment now too